eng/templates/default-build.yml

```
@@ -52,8 +52,8 @@ jobs:
       vmImage: vs2017-win2016
       ${{ if ne(variables['System.TeamProject'], 'public') }}:
         # This override makes the specified vmImage irrelevant.
-        name: NetCoreInternal-Pool
-        queue: BuildPool.Server.Amd64.VS2017
+        name: NetCore1ESPool-Svc-Internal
+        demands: ImageOverride -equals Build.Server.Amd64.VS2017
```

```suggestion
        demands: ImageOverride -equals Build.Server.Amd64.VS2019
```

Believe VS 2017 is out of support and the image doesn't exist anymore